### ISSUE TITLE
icon tap area

### ISSUE THREAD

**Issue by justinscherer** [ushahidi/platform#1885]
# Updated issue

The tap areas for icons being used as buttons are too small. Tap areas for touch-screen devices (and click areas for mouse/laptop/computer users too) should be a minimum of 40px either tall/height or wide/width. Preferably at least 40px wide and high.

See some visuals in the Figma file here: https://www.figma.com/file/Q2wVoHZ7lPllxE7zWxW19B/Ushahidi-issue-1885?node-id=0%3A1

This is making button/tap areas more accessible to people that have difficult with hand-eye co-ordination, impairments that give unsteady hand movements (like Parkinsons) and impaired sight users.

Some hit area changes are more difficult due to proximity to other clicks/taps or buttons.

Could also be combined or connected to checkbox/radio button restyling if wanted: https://github.com/ushahidi/platform/issues/2698

-------
Original issue text written by @justinscherer 
The tap areas for icons being used as buttons are too small. 

They should be increased to (tbd)px

Aha! Link: https://ushahiditeam.aha.io/features/PROD-569

**Comment by vanessa-ariel** [comment]
Hi, @Angamanga. Is it okay if I work on this issue?

**Comment by Angamanga** [comment]
@vanessa-ariel This one needs triage before its ready to work on, please check here for other issues: https://github.com/ushahidi/platform/issues?q=is%3Aopen+is%3Aissue+label%3A%22Community+Task%22

**Comment by rowasc** [comment]
@Erioldoesdesign I saw you mentioning you may want to help with design? (Thanks BTW, sorry for the delay I was sick and had a hard time keeping up with notifs) . ZERO pressure but if you do, this may be a good opportunity to fix the tap area issues? Or errr, bring clarity to what this ticket meant (or close it if we think it's not something to do??? I'm officially out of my depth)
Keeping in triage until we know more ! 
cc @Angamanga

**Comment by Erioldoesdesign** [comment]
So there general advice that tap areas for icons/buttons should be a minimum of 40px by 40px as that accounts for an 'average' finger or thumb size and also, those that might not have steady fingers/hand motion. So typically as long as something clickable in a UI is a min 40px wide and high you should be all good.

That....doesn't always mean thats easy because we all know tool tips and smaller icons exist like the familiar (i) and (?) icons in typical UI's that are tiiiiny. 

So the easy bit is editing Justin's issue to include that any clickable/tappable area should try to be min 40px by 40px.

I would say the biggest improvement that can be made is for checkboxes and radio buttons on mobile for platform. That'd be a good place to experiment with how increasing size and hit size can improve accessibility 😄 

Happy to update the issue and do some diagrams and link some background reading if that's a good way forward?

**Comment by Angamanga** [comment]
@Erioldoesdesign Yes please :) 🙏

**Comment by rowasc** [comment]
Oh wow yes that'd be fantastic thanks you

**Comment by Erioldoesdesign** [comment]
@Angamanga @rowasc it is done! 😄

**Comment by Angamanga** [comment]
@Erioldoesdesign Thank you ❤️ !!!
